tests/network/vmi_istio.go

```diff
@@ -538,6 +543,32 @@ func generateVirtualService() *unstructured.Unstructured {
 	}
 }
 
+func generateServiceEntry() *unstructured.Unstructured {
+	return &unstructured.Unstructured{
+		Object: map[string]interface{}{
+			"apiVersion": fmt.Sprintf("%s/%s", networkingIstioIO, istioApiVersion),
+			"kind":       "ServiceEntry",
```

Comment:
const ?

Reply:
I'm not sure if it's any helpful, it's only used here.